doc/release-notes-18908.md

```diff
@@ -0,0 +1 @@
+- Fixed undefined behavior error in ArgsManager::ParseParameters that appeared while running system.cpp fuzzer.
```

Comment:
This does not require release notes. It's not a user-visible change.

Reply:
Have removed the document.